docs/registry/access-mgmt-registry/README.adoc

```diff
@@ -118,14 +118,16 @@ In {product-long-registry}, you can assign user roles in your {registry} instanc
 . In the web console, go to *{registry}* > *{registry} Instances* and click the name of the {registry} instance that you want to assign roles for.
 . Click the *Access* tab to view the accounts and roles already assigned for this instance.
 . Click *Grant access* to assign roles to accounts.
-. In the *Account* field, select or enter the service account or user account name that you want to assign the role to: 
+. In the *Account* field, select or enter the service account or user account name that you want to assign the role to:
 ** A service account enables your application or tool to connect securely to your instance
 ** A user account enables users in your organization to access instances
++
+If you don't see users in the drop-down list, ask your organization administrator to grant access to view other user accounts. For more information, see {base-url}{access-mgmt-url-registry}#proc-user-account-access_managing-access-service-registry[Using RBAC to allow users to view other user accounts].
```

Comment:
Perhaps format this para as a TIP or NOTE for more prominence?

Reply:
Okay